### PR TITLE
70-80's Rework

### PR DESCRIPTION
**Pull request by Eeillios**
Finished content for the 70's, lacks only a few pieces of loc, only issues are found in the new string of code from the two coup events. Added the Seveso Disaster, elections for all the timeframe, two new terror attacks, the 77 movement, shift in the PCI from ML to Eurocommunism, Trentino autonomy and reviewed the Kidnapping of Aldo Moro with dynamic outcomes now.

### CODE REVIEW

**Comment by Eeillios** [comment]
I don't think that the Südtirol Paket should release south tyrol, the agreement OTL just gave the province of Bolzano/Alto-Adige the same degree of autonomy that Sicily, Aosta Valley and Friuli had, it just separated them from the autonomy of the province of Trento. SARs in game are used only for places that have political and judicial independence, like Hong Kong and Macau or the Bantustans or some trust territories. Turning them independent would be like making Scotland and Wales SARs after devolution. So I think that it's more accurate to represent it just as more autonomy, maybe an option for releasing should be added but that shouldn't be the historical one

**Comment by settintotrieste** [comment]
I prefer it to be a puppet to make the choice more consequential, otherwise the options become rather meaningless as any lower level of autonomy will just mean total control over the province in vicky.

**Comment by Eeillios** [comment]
Could an "autonomous region" province modifier work better? If it won't I'd rather cut the event than have the historical choice give ahistorical results and be different from any other country

**Comment by settintotrieste** [comment]
A province modifier won't cut it unfortunately. Using a puppet is the best way to represent autonomy in this sort of situation where minority pops want more control. This is as the puppet's primary culture would be different from the overlord so the pops in the region would be pacified.

**Comment by Eeillios** [comment]
Maybe adding german as accepted culture and removing austrian cores instead of autonomy? Similiarly to the Osimo Treaty

**Comment by settintotrieste** [comment]
I don't think that will be good. It will make italy too powerful by allowing them to easily annex Germany.

**Comment by settintotrieste** [comment]
@Eeillios is this branch still being worked on?

**Comment by settintotrieste** [comment]
Do you think if you can finish this branch by the end of this week? I’m planning to push a new version by the end of the year so it’d be great if I can include this branch in the update.

File: CWE/decisions/italy.txt [hunk 1]
```diff
@@ -253,4 +254,205 @@ political_decisions = {
 		}
 	}
 	
+	#1980s Reforms
```
Comment (settintotrieste) [inline CWE/decisions/italy.txt:257]: Localisation is missing in this branch

File: CWE/events/italy.txt [hunk 2]
```diff
@@ -1990,25 +1983,58 @@ country_event = {
...
+		tag = USA
 		year = 1962
 		NOT = { year = 1990 }
 		
-		has_country_flag = ITA_Enrico_Mattei
+		NOT = { OR = { government = proletarian_dictatorship government = proletarian_dictatorship1 government = socialist_democracy } } 
+		
+		NOT = { OR = { ruling_party_ideology = socialist ruling_party_ideology = progressive ruling_party_ideology = communist ruling_party_ideology = communist_social } }
+		
+		ITA = { has_country_flag = ITA_Enrico_Mattei }
```
Comment (settintotrieste) [inline CWE/events/italy.txt]: This flag is not fired by any decision or event in this branch

**Comment by Eeillios** [comment]
Yeah sure, I'm adding the Sigonella Crisis, getting all althis elections up to 1990 and a few decisions and the first phase of the rework would be fully done, the only thing I am afraid of is testing, I'll take more than a day for that and I think there are a few events that may fire when they are not supposed to, so we'll see on that front

**Comment by settintotrieste** [comment]
Okay I see, I'll delay the update to the first week of the new year so you can have more time to do the testing.

**Comment by Eeillios** [comment]
Forgot to write it in the description of the push but there also is a problem with decisions showing up twice

File: CWE/localisation/nwo_events.csv [hunk 3]
```diff
@@ -13543,13 +13543,28 @@ EVT_294_DESC;At 22:39 of the 22nd of October 1963 a landslide beginning from Mou
...
+EVT_8020348_DESC;Following the formation of the latest Andreotti government, the Prime Minister has had to resign from his place citing the protests from within the party and the ruling coalition, with the PSDI withdrawing support when MSI-DN's votes turned decisive in the last parliamentary meeting. Forced to abdicate, Andreotti is far from having left the arena, with the help of DC veteran Guido Gonella, he has been vying within the party to play the factions to secure a single party government headed by Gonella and with outsider support from the PLI and the MSI-DN, maintaining the Centre-Right shift. Shall he fail in this task it is very likely that Moro and Fanfani's factions will be able to instate the 'organic' Centre-Left with Rumor at its head.;;;;;;;;;;;;;x
+EVTDESC8020349;.;;;;;;;;;;;;;x
```
Comment (settintotrieste) [inline CWE/localisation/nwo_events.csv:13558]: Please insert localisation for these event description entries.

**Comment by settintotrieste** [comment]
Which are the decisions that are showing up twice? Also, please remember to add localisation for those events and decisions that you have not added localisation for yet.

File: CWE/events/italy.txt (hunk 2 above)
Comment (settintotrieste) [reply CWE/events/italy.txt]: I believe that this problem has yet to be fixed.

**Comment by Eeillios** [comment]
Yeah I'm working on localization, it's just that adding localization for 40 events takes a while, for the decisions, both the ones with localization and the ones with show up twice, I checked if I had accidentally inserted them twice but that wasn't the case, perhaps I need all decisions in the file to have loc?

**Comment by settintotrieste** [comment]
Alright, thanks! Perhaps you have a duplicate Italy decision file somewhere in the decisions folder? The last time I had such a problem it was because of that.

File: CWE/localisation/nwo_decisions.csv [hunk 4]
```diff
@@ -377,4 +377,8 @@ snowy_mountains_scheme_title;The Snowy Mountains Scheme;;;;;;;;;;X
...
 xea_enlarge_union_gp_title;Eurasian Union a Community of Equal Nations;;;;;;;;;;X
 xea_enlarge_union_gp;Eurasian Union difficult compromises;;;;;;;;;;X
-xea_enlarge_union_gp_desc;To join the Eurasian Union, a state needs to fulfil economic and political conditions, which require the consent of all existing members and the candidate's adoption of existing EEU laws and implementing previous decisions made by the Eurasian Economic Commission.;;;;;;;;;;X
\ No newline at end of file
+xea_enlarge_union_gp_desc;To join the Eurasian Union, a state needs to fulfil economic and political conditions, which require the consent of all existing members and the candidate's adoption of existing EEU laws and implementing previous decisions made by the Eurasian Economic Commission.;;;;;;;;;;X
+enel_title;Law 1643;;;;;;;;;;X
```
Comment (settintotrieste) [inline CWE/localisation/nwo_decisions.csv:381]: For decision titles please use this style: Enact/ Establish X etc

So enel_title;Law 1643 should be "Enact Law 1643"

Same for iri_prodi_title;IRI Restructuring should be "Restructure the IRI"

File: CWE/localisation/nwo_decisions.csv [hunk 5]
```diff
@@ -377,4 +377,14 @@ snowy_mountains_scheme_title;The Snowy Mountains Scheme;;;;;;;;;;X
...
+agensud_title;Agency for Promotion and Development of the South;;;;;;;;;;X
+agensud_desc;Born with Law 488, the Agency for Promotion and Development of the South (AgenSud for short) replaced the Fund for the South established during the 50's and dissolved by decree with Law 64 earlier this year. The main change, besides the namechange, lays in the modernization of the Fund to fit with the new European Cohesion Policies, where instead of allocating only resources and funds, function now mostly overtaken by the ministry of economy and finance, the agency will provide services and means to incease the human and social capital in the region, to allow for greater partecipation of southerners, who in the last decade flocked in thousands to the north due to the dire situation, into the national economy and politcs.;;;;;;;;;;X
+madama_agreement_title;Agreement for the Revision of the Lateran Concordat;;;;;;;;;;X
```
Comment (settintotrieste) [inline CWE/localisation/nwo_decisions.csv:387]: This should be "Revise the Lateran Concordat"

File: CWE/localisation/nwo_decisions.csv [hunk 6]
```diff
@@ -377,4 +377,14 @@ snowy_mountains_scheme_title;The Snowy Mountains Scheme;;;;;;;;;;X
...
+madama_agreement_desc;On the example of the Second Vatican Council, the Church and the Italian state have long been trying to review the 1929 Lateran Treaty that ties Rome to the Vatican since the 1948 Constitution became law. Whast has been dubbed as the Concordat Bis, will guarantee the separation of the church and state, removing catholicism as official religion of Italy, ensuring freedom from taxation and perquisition for the church, the recognition of religious holidays and the abolition of the congrua, an old tax on all citizens that went directly to the Catholic Church, in favour of a multiple choice tax that can be given to any registered religious organization.;;;;;;;;;;X
+grande_riforma_title;The Great Reform;;;;;;;;;;X
```
Comment (settintotrieste) [inline CWE/localisation/nwo_decisions.csv:389]: This should be "Begin The Great Reform"

**Comment by Eeillios** [comment]
If you can't find any issue I think the pull is ready to be merged, all that was meant to be in is and even a few extras, only issue is that the game won't start as of now but that might be just an issue with my launcher rather than an actual problem with the additions

**Comment by settintotrieste** [comment]
Thanks!

**Comment by settintotrieste** [comment]
I've managed to fix the CTD to desktop.

**Pull request merged**